dit/dit_exporter.py

```diff
@@ -191,6 +191,7 @@ def task(group, group_id, subgroup, subgroup_id, task, task_id, data):
         _overall_time_spent += time_spent
 
         if statussing and not verbose:
+            first_entry = logbook[0]
```

Comment:
This may not be the actual first entry because the `logbook` variable comes from the filtered `data` which may not contain all logbook entries.

So you need to either make sure this is never the case (with tests?) or take this from the unfiltered data.